docs/how-to/working-with-rocfft.rst

```diff
@@ -344,7 +347,7 @@ The workflow for fields is as follows:
 #. Create the plan by calling :cpp:func:`rocfft_plan_create`.  Pass the plan
    description that has already been allocated.
 
-#. Execute the plan by calling :cpp:func:`rocfft_execute`.  This function takes
+#. Execute the plan by calling :cpp:func:`rocfft_execute`.  This function accepts
```

Comment:
This function accepts...

The change from _takes_ to _accepts_ seems to imply that it will accept but not require arrays of pointers. Does it take something other than arrays of pointers? I am just concerned that simply by changing takes to accept you have not actually communicated anything.

Reply:
I see from the next couple of paragraphs, that the accepting of arrays of pointers for input and output specifically applies to  distributed transforms. I would make that more clear: 

This function accepts arrays of pointers. For in-place transforms, the arrays of pointers should be provided for the inputs only. For distributed transforms, the arrays of pointers include both inputs and outputs.

Reply:
I've reverted the takes -> accepts change.  I think I started out rewriting that sentence but then changed my mind and didn't quite undo everything.

The rest I think is only a concern when "distributed" is treated as the opposite of "in-place".